README.md

```diff
@@ -262,7 +244,7 @@ NOTE : You can find the SQL script [here](./resources/database_initializer.sql).
 - Navigate to `managing-database-transactions/guide` and execute the following command in a terminal to run this sample.
 
 ```bash
-   $ ballerina run banking_application
+   $ ballerina run --experimental banking_application
```

Comment:
Shall we add why we use --experimental flag, just saying Ballerina transactions are at experimental stage.